index.html

```diff
@@ -10,9 +10,18 @@
 </head>
 <body>
   <header class="navbar">
-  <nav class="container">
+    <div class="nav-logo">
     <a  class="logo" href="#">My logo</a>
-    <img class="mobile-menu" src="img/Union.png" alt="open navigation">
+  </div>
+  <nav class="container">
+    <div class="menu-wrapper">
+      <img class="mobile-menu" src="img/Union.png" alt="open navigation">
+    </div>
+    <ul class="nav-list">
+      <li class="nav-item">Portfolio</li>
+      <li class="nav-item">about</li>
+      <li class="nav-item">contact</li>
+    </ul>
   </nav>
 </header>
 <section class="hero">
```

Comment:
- [ ] Kindly design the hero section following the Figma template for the mobile and desktop version
- Figma design
<div style="display:inline-flex;">
<img width="440" alt="Screen Shot 2022-03-25 at 15 34 53" src="https://user-images.githubusercontent.com/28307960/160132802-ba235a25-91ff-41e2-9662-748aaa808cae.png">
<img width="268" alt="Screen Shot 2022-03-25 at 15 38 27" src="https://user-images.githubusercontent.com/28307960/160132732-1f112751-60af-48a2-b217-15b2c25d7307.png">
</div>
- Your design
<div style="display:inline-flex;">
<img width="440" alt="Screen Shot 2022-03-25 at 15 35 13" src="https://user-images.githubusercontent.com/28307960/160133117-0dcc766d-45ea-483b-8d8a-4c6d207adfeb.png">
<img width="268" alt="Screen Shot 2022-03-25 at 15 38 11" src="https://user-images.githubusercontent.com/28307960/160133066-31807c00-478f-4cc7-9a06-5b98f61bd95b.png">
</div>